schemas/v1/20_actor_add_state.go

```diff
@@ -0,0 +1,17 @@
+package v1
+
+func init() {
+	patches.Register(
+		20,
```

Comment:
```suggestion
		21,
```

Don't forget to rename the file as well from `20_actor_add_state.go` to `21_actor_add_state.go`